exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -1138,7 +1185,7 @@ func (ku *Kucoin) GetOrderHistory(ctx context.Context, getOrdersRequest *order.G
 		log.Errorf(log.ExchangeSys, "%s %v", ku.Name, err)
 	}
 	order.FilterOrdersByPairs(&orders, getOrdersRequest.Pairs)
-	return orders, nil
+	return getOrdersRequest.Filter(ku.Name, orders), nil
```

Comment:
From [previous nit](https://github.com/thrasher-corp/gocryptotrader/pull/1102#discussion_r1050394134) look at the definition of the filter methods. You will be able to remove the calls above.